mav_msgs/include/mav_msgs/common.h

```diff
@@ -149,6 +149,84 @@ inline void skewMatrixFromVector(const Eigen::Vector3d& vec,
       0.0f;
 }
 
+inline void vectorFromSkewMatrix(const Eigen::Matrix3d& vec_skew,
+                                 Eigen::Vector3d* vec) {
+  assert(vec);
+  if ((vec_skew + vec_skew.transpose()).norm() < 1e-6){
```

Comment:
how's 1e-6 chosen? In any case I think it would be good to have this as a constant defined somehwere

Reply:
(I think it's a bit large)

Reply:
Setting small value check to 1e-6, matlab isskew.m function uses 1e-5